### DISCUSSION TITLE
Need Help for renewing GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by skyone-wzw** [community/community#47869] [GitHub Education] [answered]
Hello! I am a university student. I applied for and was approved for the GitHub Student Developer Pack in 2021. Yesterday, I received an email from GitHub reminding me to renew my GitHub Student Developer Pack. When I tried to renew according to the instructions at https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#expiration-and-renewals

```
xxxx@xxxx.edu.cn has already been used to obtain an academic discount on GitHub account(s) skyone-wzw.
```

And the website prompted me to provide student identification and other information again (just like when I first applied for the GitHub Student Developer Pack). As far as I remember, this was not required when renewing in 2022. I am concerned that if I continue to fill out the form, it may be mistaken as a new application for the GitHub Student Developer Pack (rather than a renewal). Therefore, I would like to know whether I should wait for the GitHub Student Developer Pack to expire before renewing, or continue filling out the form with my student identification and other information.

**Comment by melvincwng** [comment]
@skyone-wzw, to my knowledge, GitHub Education should renew your GitHub Student Developer pack as long as you provide the necessary documents to reverify your academic affiliation.

If any issue arises during the renewal process,  you can also raise a ticket and contact GitHub Education Support over [here](https://support.github.com/contact/education).

Hope this helps and answers your question 😃

**Reply by ghost** [reply]
ต่้องการต่ออายุ

**Comment by samus-aran** [comment, marked as answer]
Hi @skyone-wzw  👋

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.
We apologize for any inconvenience this may cause.
To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Thank you for your understanding.

**Reply by skyone-wzw** [reply]
Thank you very much for your answer.

**Reply by sofiamariano** [reply]
Hello! I have the same issue but when I access https://education.github.com/benefits I can't find where to submit my information again, could you help me?

**Reply by skyone-wzw** [reply]
> Hello! I have the same issue but when I access https://education.github.com/benefits I can't find where to submit my information again, could you help me?

The documentation in [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#expiration-and-renewals) is very clear and includes a screenshot. Just open https://education.github.com/discount_requests/application

**Reply by sofiamariano** [reply]
![Screenshot 2023-06-20 ](https://github.com/community/community/assets/106205749/4a64997c-8cdd-4dd3-93e8-3b22fa48073b)

Thanks for helping, but the issue is that I can't click on continue to finish my application

**Reply by mende1** [reply]
@sofiamariano Yeah, I'm facing the same problem. Have you had any updates?

**Reply by CoolGenius-123** [reply]
Hello I have been Accessing GitHub Student Pack from Last 2 years but now My Github Student got Expired, so I need to do the reverification since I am still the student whose 1.5 yrs still left in my completion of my course and our college don't provide any kind of student email so I am applying from my normal email which I have used before for accessing the Student pack but now its rejecting it saying we need you to access using Student email So Help me what can I do .

**Reply by aayushchugh** [reply]
@CoolGenius-123 I am facing same problem, was you able to solve this?

**Reply by Vigneshgvs** [reply]
Update: https://education.github.com/discount_requests/application (scroll down) and fill out the form. Will work, if you upload ID with date or "Student enrollment verification" form ur university (all univ do provide online).
@skyone-wzw Will the option "Reverify your academic affiliation" as in the screenshot - appear under our profile only after the current plan expires? Can't renew before the plan with the code/offer of being a student expires?
Because I can't see the "Reverify your academic affiliation" option anywhere, and mine is going to expire in a few days.

**Reply by anguyen1776** [reply]
Hey @samus-aran , my account was approved but I'm still unable to access the pack.

**Reply by samus-aran** [reply]
Hey @anguyen1776 , my account was mentioned but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍 😉 

Better yet check out our updated [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. 🎉

**Comment by nirmal-k-r** [comment]
Hi, I am encountering an issue, I've resubmitted it with 2 different university email but it's stuck on continue. Ive tried it a few weeks ago and now but it's still stuck.


Anything I've missed?

**Reply by aidarbekq** [reply]
Hi, did you find a solution?

**Comment by Brianceasar** [comment]
Hi, I am encountering an issue. When reverifying I'm required to use my university email. The university i go to do not provide emails. What can I do

**Reply by 2Sourav** [reply]
Same with me. I too do not have any university-provided emails. What to do now?

**Reply by CoolGenius-123** [reply]
Have you got any solutions to this I am also facing same issues

**Comment by nomehagasmuchocaso** [comment]
Same issue, my learning center do not provide edu mail and i cant re-verify my student status, what can i do?

**Comment by massidi** [comment]
I am encountering an issue is someone get the solution?

**Comment by Mahrkeenerh** [comment]
The /benefits page is now changed to something else, so the existing solutions don't help anymore ...

**Reply by Vigneshgvs** [reply]
https://education.github.com/discount_requests/application (scroll down) and fill out the form. Will work, if you upload ID with date or "Student enrollment verification" form ur university (all univ do provide online).

**Reply by Mahrkeenerh** [reply]
Yes, that's the one!

**Comment by Taoshib** [comment]
Recently I applied for re-verification. My application got approved . But I am still not being able to access my pack. A message saying that I am in a waiting period. Can anybody tell me how long is it going to take?

**Comment by artysan-code** [comment]
I'm having the same problem, I had an account called rgoth-main and it cannot receive the discount for "already had a discount for this email" even if I could give proofs of my actual subscription to university. I created a new account and I'm facing the same issue, telling it has been used on the old account but I resubscribed to university.

**Comment by ericcwlin** [comment]
I got an email said: To reverify, simply return to https://education.github.com/benefits and click "submit your information again."

But there aren't any sections that have "submit your information again"

**Reply by echoface** [reply]
same here

**Comment by echoface** [comment]
something going wroing, all student relative page redirect to "https://github.com/pricing" page, and no "submit your information again" found

**Reply by jcjr30** [reply]
Mine as well

**Reply by TengBoYuu** [reply]
Did you solve it? I had the same problem.

**Reply by ghost** [reply]
Same for months. Github Support isn't replied for 3 times. It's a so bad situtation. If you find solution please share with us.

**Reply by shiqingtanya** [reply]
Found solution?

**Comment by minhazulislam** [comment]
I received an email informing me that I need to enter my payment information because GitHub Pro will start charging me this month. I've been using the GitHub Education Student Pack, and I haven't graduated from school yet. Does anyone have a solution for this issue? Additionally, when I tried to submit my school information to renew the plan, the [website](https://education.github.com/discount_requests/application) froze at the "Continue" button.

**Reply by aslamanver** [reply]
Found solution?

**Reply by minhazulislam** [reply]
Yes. I found the solution here - https://github.com/orgs/community/discussions/111330#discussioncomment-12617825.
